jetty9-base/src/main/jetty-base/config-scripts/jetty.commands

```diff
@@ -0,0 +1,11 @@
+--create-startd
+
+--add-to-start=server,webapp,http,deploy,jsp,jstl,resources
+
+jetty.httpConfig.outputAggregationSize=32768
```

Comment:
I don't think they are really necessary anymore.  I think they are hangovers from when logging action was taken after commit.    The image should not now care and can probably use the default.